pkgs/development/r-modules/default.nix

```diff
@@ -955,6 +955,13 @@ let
     });
 
     xml2 = old.xml2.overrideAttrs (attrs: {
+      patches = [
+        (fetchpatch2 {
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions

please add a comment to the patch